### PR TITLE
build: upgrade to v1.0.2

### PR DESCRIPTION
**Pull request by montesmariana**
We could still add the logger tests here before merging

### CODE REVIEW

File: tutorial/README.md [hunk 1]
```diff
@@ -8,14 +8,12 @@ This tutorial shows you
...
 
-You can install the package and it's dependencies with `pip`:
+You can install the package and its dependencies with `pip`:
 
 ```python
-pip install "git+https://github.com/kuleuven/mango-mdschema@1.0.1"
+pip install mango-mdschema
 ```
 
-> The package will be published to [Pypi](https://pypi.org/) in the future.
```
Comment (rmoreas) [inline tutorial/README.md:17]: This should (also) be removed in the Python notebook. The notebook should be the "master" document to create the markdown version.

Comment (rmoreas) [reply tutorial/README.md:17]: Rest looks good to me! Thanks.

**Review by rmoreas — CHANGES_REQUESTED** [review]

File: tutorial/README.md (hunk 1 above)
Comment (montesmariana) [reply tutorial/README.md:17]: Thank you! I thought I had updated it. I alsor re-rendered the README.md but the result is the same so there is no diff.

Comment (rmoreas) [reply tutorial/README.md:17]: 💯

**Pull request merged**